docs/source/getting_started/installation.rst

```diff
@@ -26,6 +26,10 @@ You can install vLLM using pip:
     $ # Install vLLM with CUDA 12.1.
     $ pip install vllm
 
+.. note::
+
+    Although we recommend using `conda` to create and manage Python environments, it is highly recommended to use `pip` to install vLLM. This is because `pip` can install `torch` with separate library packages like `NCCL`, while `conda` installs `torch` with statically linked `NCCL`. This can cause issues when vLLM tries to use `NCCL`. See `this issue <https://github.com/vllm-project/vllm/issues/8420>`_ for more details.
+
```

Comment:
We need to either use \`\`text\`\` or :code:\`text\` for code blocks in reStructuredText.

Reply:
nice catch! I fixed some previous code text as well.